force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +88,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> wordsList = sentence.split(' ');
         // Create a set to store the unique words
-
+        Set<String> wordSet = new Set<String>();
         // Add each word to the set
-
+        for (String word:wordsList) {
+            wordSet.add(word);
+        }
         // Join the unique words back into a sentence
-
+        List<String> uniqueWordsList = new List<String>();
+        uniqueWordsList.addAll(wordSet);
+        String result = String.join(uniqueWordsList,' ');
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
```

Comment:
Good, but you made it a little harder on yourself than it needed to be. You can pass a List directly into a Set constructor and skip  the loop

```
public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
```